bip-0179.mediawiki

```diff
@@ -0,0 +1,56 @@
+<pre>
+  BIP: 179
+  Title: Name for payment recipient identifiers
+  Author: Emil Engler <me@emilengler.com>
+          MarcoFalke <falke.marco@gmail.com>
+          Luke Dashjr <luke+bip@dashjr.org>
+  Status: Draft
+  Type: Informational
+  Created: 2019-10-17
+  License: CC0-1.0
+</pre>
```

Comment:
Missing Comments-URI in bip-0179.mediawiki at scripts/buildtable.pl line 176, <$F> line 11.

Reply:
Done, see https://github.com/bitcoin/bips/wiki/Comments:BIP-0179

Reply:
Please add this to the readme:

```diff
+> | [[bip-0179.mediawiki|179]]

+> |

+> | Name for payment recipient identifiers

+> | Emil Engler, MarcoFalke, Luke Dashjr

+> | Informational

+> | Draft

+> |-

Reply:
Done